services/shuffle/neff/handler.go

```diff
@@ -29,7 +27,7 @@ import (
 	"golang.org/x/xerrors"
 )
 
-const watchTimeout = time.Second * 2
+const watchTimeout = time.Second * 6
```

Comment:
is there any justification for 2 or 6 ? 😅

Reply:
A reasonable amount of time after which you can assume Dela won't include your transaction ... 😁
I've relaxed it to accommodate for the integration test. The timing is a real question though, that is far from trivial. There are a lot of factors (number of nodes, network, CPU power, election and consensus parameters, etc...) that should be taken into account to be able to "formally" compute it. Right now we go with the "good enough" solution.